pyrobuf/protobuf/templates/pyrobuf_list_pyx.tmpl

```diff
@@ -227,9 +196,8 @@ cdef class StringList(list):
         else:
             raise TypeError("list indices must be integers, not %s" % type(i).__name__)
 
-        listener = <object>self._listener
-        if not listener._is_present_in_parent:
-            listener._Modified()
+        if self._listener is not None:
```

Comment:
To clean things up a little, we could use a noop listener as the default and skip the `if` statement, but this is fine.